### DISCUSSION TITLE
Multi-Tenancy & Namevaluelist

### DISCUSSION THREAD

**Discussion by ashishsinha24** [MarimerLLC/csla#2810] [Questions] [answered]
Hi,

I am developing a multi-tenant app using CSLA 6.0. Each tenant has its own specific Nvls
What are the best practices to share the CachedList (which is static) in a NVL ?

**Comment by rockfordlhotka** [comment, marked as answer]
I'd look at something like REDIS cache, as it provides a lot of services that are useful in more complex caching scenarios.

Alternately, you can use a static dictionary where the key includes the tenant id and type or something like that.
